README.md

```diff
@@ -1,5 +1,35 @@
 # TranslateX
-An Android App based on NLP(Natural Language processing) to translate text from one language to another
-
-It will use Firbase ML Kit's "On Device Translation"
+## Table of Contents
+Getting Started
+Objective
+Creating issues
+Already created issues
+Labels
+App icon
+Deploying bots
+Resources
+Rules and Scoring System
+## Getting Started
+TranslateX is an Android App based on the concept of NLP(Natural Language processing).
+## Objective
+The objective of this application is to translate text from one language to another.
+## Creating Issues
+For creating issues, follow the template provided in the link below:
+https://github.com/Mehulipal/TranslateX/tree/development/.github/ISSUE_TEMPLATE
+## Already Created Issues
+Some issues have already been created. To claim one,visit the link given below:
+https://github.com/NJACKWinterOfCode/TranslateX/issues
+## Labels
+The issues provided in the link above have some labels like 'Beginner', 'Intermediate' to help you choose the issue that matches your comfort level.
+## App Icon
+To design an icon for this app, follow the link below:
+https://github.com/NJACKWinterOfCode/TranslateX/issues/6
+## Deploying Bots
+A special issue has been created for this purpose. Follow the link below:
+https://github.com/NJACKWinterOfCode/TranslateX/issues/7
+## Resources
+This app uses Firbase ML Kit's "On Device Translation"
 For more ,see https://developers.google.com/ml-kit/language/translation
+## Rules and Scoring System
+This project follows the rules and guidelines of NJACKWinterOfCode for allotting points. Check out the link below:
+https://github.com/NJACKWinterOfCode/Get-Started#rules-and-scoring-system
```

Comment:
Also mention about the main libraries and the API's used.

Reply:
I have done the changes,can you please review it?